server/repository/src/migrations/v2_10_00/add_purchase_order_tables.rs

```diff
@@ -89,7 +89,8 @@ impl MigrationFragment for Migrate {
                     stock_on_hand_in_units {DOUBLE} NOT NULL DEFAULT 0.0,
                     supplier_item_code TEXT,
                     price_per_unit_before_discount {DOUBLE} NOT NULL DEFAULT 0.0,
-                    price_per_unit_after_discount {DOUBLE} NOT NULL DEFAULT 0.0
+                    price_per_unit_after_discount {DOUBLE} NOT NULL DEFAULT 0.0,
+                    comment TEXT
```

Comment:
Personally think that we should write another migration for this. Though it's only in develop and no user should have used this yet so its fine to change this? Thoughts @jmbrunskill

Reply:
Yes I think a new migration for comment would be better. Changing to not null is ok if we have always had values there should be fine, but a  new column everyone will miss, so better to do a new migration